src/RTTComms.cpp

```diff
@@ -76,21 +76,22 @@ namespace BrainCloud
 
     void RTTComms::resetCommunication()
     {
-        if (_isConnected)
+        if (isRTTEnabled())
         {
-            _isConnected = false;
+            _rttConnectionStatus = RTTConnectionStatus::Disconnecting;
```

Comment:
ok I was wondering why this state even exist, but ya I think that's safe in case something happen with the events. nice one